isis/src/control/objs/ControlNetVitals/ControlNetVitals.cpp

```diff
@@ -223,23 +225,23 @@ namespace Isis {
     QString details = "";
     if (hasIslands()) {
       status = "Broken!";
-      details = "This network has " + toString(numIslands()) + " islands.";
+      details = "This network has " + toString(numIslands()) + " island(s).";
```

Comment:
If this happens, there will always be at least 2 islands, so we can just always use the plural form.